### ISSUE TITLE
Cannot scrobble to Last.fm

### ISSUE THREAD

**Issue by Crazyphil** [vhanla/GMusicDesktopPlayer#2]
When opening the Options dialog, there is no option to connect GMusic to a Last.fm account:
![Screenshot](https://cloud.githubusercontent.com/assets/3602847/20309951/56a97ca0-ab4a-11e6-86f1-0d7407741974.png)

**Comment by vhanla** [comment]
Thank you for your feedback, button for LastFM authorization was accidentally set to transparent. If you move the mouse cursor above 'LastFM User:' it will change the cursor's icon. 

I attached a fix in the Release section as well the commit in the source code.

[Fixed executable](https://github.com/vhanla/GMusicDesktopPlayer/releases/tag/1.9.27b)

**Comment by Crazyphil** [comment]
Thanks for the quick fix. I can see and click the Authorize button now, but unfortunately, nothing happens after the text changes to "Please wait...".

I see in the source code that this process seems to use Pascal methods for connecting to the internet instead of those of the browser engine. Since I have to use a proxy server to get outside the firewall, I suspect that these methods don't regard the system proxy settings. Am I right in this assumption?

**Comment by vhanla** [comment]
You are right, it is a pascal code, and no proxy support was added, but I plan to add it. Thanks again for your feedback.
